src/main/resources/avro/variants.avdl

```diff
@@ -61,6 +94,15 @@ record VariantSet {
 }
 
 /**
+Some CallSets reflect genotype calls, while others reflect linked haplotype
+calls.
+*/
+enum CallSetType {
+  GENOTYPE,
+  HAPLOTYPE
+}
+
+/**
 A `CallSet` is a collection of variant calls for a particular sample.
 It belongs to a `VariantSet`. This is equivalent to one column in VCF.
```

Comment:
A `CallSet` can span `VariantSet`s, for talking about situations in which you have fed the same data to different callers and gotten different results.

I'll change the comment about it "belonging" to a `VariantSet` to something more correct, especially given the existence of two `CallSetType`s.